templates/organization/home.html.twig

```diff
@@ -27,7 +27,7 @@
 
     {% if app.user.isParent() %}
         <hr>
-        <p><a class="btn btn-danger" href="{{ path('app_organization_planning', {minimumAvailableHours: 1, hideAssets: 1}) }}" role="button">{{ 'organization.showAllUsersAvailability' | trans }}</a></p>
+        <p><a class="btn btn-danger" href="{{ path('app_organization_planning', {minimumAvailableHours: 1, hideAssets: 1, 'userPropertyFilters[vulnerable]': 0}) }}" role="button">{{ 'organization.showAllUsersAvailability' | trans }}</a></p>
```

Comment:
this will be buggy once user properties will be "really" dynamic

Reply:
It's not really an issue, as the form is ignoring unknowns values